subsys/net/lib/nrf_cloud/src/nrf_cloud_pgps.c

```diff
@@ -1778,6 +1782,8 @@ int nrf_cloud_pgps_init(struct nrf_cloud_pgps_init_param *param)
 		}
 	}
 
+	state = PGPS_INITIALIZED;
```

Comment:
what is the purpose of this new state?
`PGPS_INITIALIZING` and `PGPS_INITIALIZED` are set but never checked for.
would a single state of `PGPS_INIT` cover both?

Reply:
Removed and went back to previous.